02-oop-inheritance/trackObserver.js

```diff
@@ -0,0 +1,5 @@
+//Constructor Function
+TrackObserver  = function(event){};
```

Comment:
What do an observer has ? It should have multiple event -> callback pairs. If you need some explanation please let me know